docs/jsonapi-spec.textile

```diff
@@ -363,6 +448,15 @@ h4(#commandFindOptions). find Command Options
 | @pageState@ | ASCII String | The page state of the previous page, when supplied the @find@ command will return the next page from the result set. If unspecified, null, or an empty string the first page is returned.|
 | @skip@ | Positive Integer | Skips the specified number of documents, in sort order, before returning any documents.|
 
+h4(#commandFindResponse). find Command Response
+
+|_. Response Element |_. Description |
+| @data@ | Present with fields : @data@, @count@ and @nextPageState@ which will be @null@ if no further data is available. |
```

Comment:
* first should `@data@` be `@docs@` here?
* second, `nextPageState` will be null if there are no more documents, but count and docs should never be null.. These should be empty array and zero in case nothing is found?

Reply:
good catch on the data, and bad writing from my. Intention was only to say @nextPageState@ is null . Fixing.